src/net_processing.cpp

```diff
@@ -317,12 +317,10 @@ struct CNodeState {
     bool fPreferredDownload;
     //! Whether this peer wants invs or headers (when possible) for block announcements.
     bool fPreferHeaders;
-    //! Whether this peer wants invs or cmpctblocks (when possible) for block announcements.
-    bool fPreferHeaderAndIDs;
-    /**
-      * Whether this peer will send us cmpctblocks if we request them.
-      */
-    bool fProvidesHeaderAndIDs;
+    /** Whether this peer wants cmpctblocks (when possible) for block announcements. */
+    bool m_sendcmpct_hb{false};
```

Comment:
IMO `m_prefercmcpt`  better, otherwise name collusion is too close.

Reply:
`m_prefercmpct` is not as precise. The peer might 'prefer' to fetch blocks using getdata(CMPCT), which is not what this means. `m_sendcmpct_hb` unambiguously means "send blocks to this peer using BIP 152 high bandwidth mode".

Reply:
Could you add the "high bandwidth" part of that to the comment so the _hb makes more sense?